lib/environment.js

```diff
@@ -5,21 +5,24 @@ module.exports = {
     require: {
         // builtins required by the libraries exposed for sandbox users
         events: {preferBuiltin: true, glob: true},
+        path: {preferBuiltin: true, glob: true},
```

Comment:
required by mime-db, which in turn is required by mime-type :-/ but this is lightweight and has no side effects